roundabout-core/src/main/resources/test/roundabout.xml

```diff
@@ -1,51 +1,79 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<roundabout>
+<model>
     <!-- global parameters needed for simulation -->
     <parameters>
-        <parameter name="SIMULATION_SEED" value="1" />
-        <parameter name="MIN_DISTANCE_FACTOR_BETWEEN_CARS" value="0.1" />
-        <parameter name="MIN_DISTANCE_FACTOR_BETWEEN_CARS" value="1.0" />
-        <parameter name="MIN_TIME_BETWEEN_CAR_ARRIVALS" value="3.0" />
-        <parameter name="MAX_TIME_BETWEEN_CAR_ARRIVALS" value="10.0" />
-        <parameter name="MAIN_ARRIVAL_RATE_FOR_ONEWAYSTREETS" value="1.0" />
+        <parameter name="SIMULATION_SEED" value="1"/>
```

Comment:
thats nice, but is it also possible to create a random simulation (random seed)?